crates/bevy_internal/src/lib.rs

```diff
@@ -1,3 +1,8 @@
+#![warn(missing_docs)]
+//! This module contains plugins for quickly setting up a Bevy app, as well as publicizing of the subcrates Bevy contains
+//!
+//! This module is separated into its own crate to enable simple dynamic linking for Bevy
```

Comment:
This an internal crate that shouldn't directly be used. The `bevy` crate should instead be used.

Reply:
Thanks for pointing that out, I updated the crate doc to reflect this